src/web/mjs/connectors/ManyoonKR.mjs

```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class ManyToon extends WordPressMadara {
```

Comment:
The classname `ManyToon` is alredy used.

Reply:
There seems to be a typo in the filename too